app/src/main/java/com/duycuong/weather/ui/screen/main/nextdays_weather/ItemWeatherViewModel.java

```diff
@@ -0,0 +1,42 @@
+package com.duycuong.weather.ui.screen.main.nextdays_weather;
+
+import android.databinding.BaseObservable;
+
+import com.duycuong.weather.R;
+import com.duycuong.weather.data.model.Datum;
+import com.duycuong.weather.utils.Constant;

```

Comment:
ERROR: Unused import: com.duycuong.weather.utils.Constant. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports